src/main/java/edu/harvard/iq/dataverse/DatasetFieldValueValidator.java

```diff
@@ -168,7 +169,20 @@ public boolean isValid(DatasetFieldValue value, ConstraintValidatorContext conte
 
         if (fieldType.equals(FieldType.URL) && !lengthOnly) {
             try {
-                URL url = new URL(value.getValue());
+                
+                String[] schemes = {"http","https"};
```

Comment:
Other than being less popular, is there any reason ftp URLs shouldn't be allowed?

Reply:
Nope, have added ftp in.